### DISCUSSION TITLE
Not showing all rows in response

### DISCUSSION THREAD

**Discussion by Hrishikesh007788** [sinaptik-ai/pandas-ai#432] [Q&A]
I used a csv file having 9000+ rows. When I ask for a response, it doesn't show all rows and uses dots. What should I do if I want to see all the rows?

**Comment by gventuri** [comment]
Hi @Hrishikesh007788 thanks for asking. Could you copy/paste the code you are using? Maybe it's filtering out some rows as part of the query. Let me know!

**Reply by Hrishikesh007788** [reply]
Here is the log of what's happening the the background. I am trying to see the list of customers but its giving me the answer in this form. The output has 521 rows but its not showing everything. Is there a way to display every row? Since I am displaying the "Answer:" as it is in the frontend, this type of output is not much useful. 




2023-08-08 17:55:15 [INFO] Question: Customers generating revenue more than 10000 in FY20 Q3
2023-08-08 17:55:15 [INFO] Running PandasAI with openai LLM...
2023-08-08 17:55:15 [INFO] Prompt ID: 799acb40-b36a-4e02-b2b3-c6d0c185164b
2023-08-08 17:55:15 [INFO] Using cached response
2023-08-08 17:55:15 [INFO] 
Code running:
```
df[df['FY20_Q3_REVENUE'] > 10000]['Customer']
```
2023-08-08 17:55:15 [INFO] Answer: 49        Customer 50
57        Customer 57
84        Customer 82
85        Customer 83
98        Customer 93
            ...      
8990    Customer 8467
8998    Customer 8475
9010    Customer 8487
9021    Customer 8496
9050    Customer 8525
Name: Customer, Length: 521, dtype: object

**Reply by mspronesti** [reply]
To solve this one simply needs to add on top `pd.set_option('display.max_rows', None)`. @gventuri Do you think it is worth it to make this a standard behavior ?

**Reply by gventuri** [reply]
@mspronesti yep, 100% agree. Just wondering at which point we'd better change the behavior. As we initialize PandasAI?

**Reply by mspronesti** [reply]
We might even just put the following two lines on top of the generated code, after polishing it
```py
pd.set_option('display.max_columns', None)
pd.set_option('dipslay.max_rows', None)
```
Not sure this is a desired behavior though. Is it helpful to visualize a big dataframe in the console?

**Reply by Hrishikesh007788** [reply]
I am displaying it in the frontend. Is there a way to make it more presentable? Dataframe looks too raw in the frontend
